models/Movie.model.js

```diff
@@ -0,0 +1,15 @@
+const { Schema, model } = require('mongoose');
+
+const movieSchema = new Schema(
+  {
+    title: String,
+    genre: String,
+    plot:  String,
+    cast: {
```

Comment:
this relationship is one to many. it means you need to save array of celebrities here"

` cast: [{
      type: Schema.Types.ObjectId,
      ref: "Celebrity"
    }] `